modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -2560,6 +2511,39 @@ else if (Objects.equals(
 		}
 	}
 
+	private void _assertLocalizedValues(
+			Map<String, Object> expectedValues, long objectDefinitionId,
+			long objectEntryId, List<String> languageIds)
+		throws Exception {
+
+		for (String languageId : languageIds) {
+			for (ObjectField objectField :
+					_objectFieldLocalService.getLocalizedObjectFields(
+						objectDefinitionId)) {
+
+				Map<String, String> localizedValues =
+					(Map<String, String>)expectedValues.get(
+						objectField.getI18nObjectFieldName());
+
+				String value = GetterUtil.get(
+					localizedValues.get(languageId), "");
+
+				expectedValues.put(objectField.getName(), value);
```

Comment:
I did not understand why we edit the expectedValues. Could you reach me and explain it, please? :smile: